Assets/MixedRealityToolkit.SDK/Experimental/HandCoach/Scripts/MoveToTarget.cs

```diff
@@ -0,0 +1,197 @@
+﻿using System.Collections;
+using System.Net.NetworkInformation;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.UI
+{
+    /// <summary>
+    /// This class provides functionality to move the hand hint from a tracking position to a target position over time.
+    /// </summary>
+    public class MoveToTarget : MonoBehaviour
+    {
+        [Tooltip("Object to track.")]
+        [SerializeField]
+        private GameObject trackingObject = null;
+
+        /// <summary>
+        /// Object to track.
+        /// </summary>
+        public GameObject TrackingObject
+        {
+            get
+            {
+                return trackingObject;
+            }
+            protected set
+            {
+                trackingObject = value;
+            }
+        }
+
+        [Tooltip("Target to move to.")]
+        [SerializeField]
+        private GameObject targetObject = null;
+
+        /// <summary>
+        /// Target to move to.
+        /// </summary>
+        public GameObject TargetObject
+        {
+            get
+            {
+                return targetObject;
+            }
+            protected set
+            {
+                targetObject = value;
+            }
+        }
+
+
+        [Tooltip("Shared parent between tracking and target objects used for relative local positions.")]
+        [SerializeField]
+        private GameObject rootObject = null;
```

Comment:
I don't fully understand what this field is for...is the structure of the hand coach that you need an object to move to, and also the parent of that object? If so, why not just use targetObject.parent?

Reply:
We could do that, but the idea was that in case the user wants to target a GameObject external to the hand coach hierarchy without a direct shared parent with the hand coach, they could still configure the movement behavior correctly

Reply:
Fair enough.